data-migrations/2025-11-19-1427-init-stored-procs.sql

```diff
@@ -70,12 +65,11 @@ END;
 
 START TRANSACTION;
 
--- works: insert new works (DOIs) and skip rows that would violate the doi unique constraint
 INSERT IGNORE INTO works (doi)
 SELECT doi
 FROM stagingWorkVersions;
 
--- workVersions: insert new work versions
```

Comment:
I find the comments helpful for getting an overview about what each section does, especially after not reading it for a while.